src/nile/nre.py

```diff
@@ -25,9 +24,15 @@ def compile(self, contracts):
         """Compile a list of contracts."""
         return compile(contracts)
 
-    def declare(self, contract, alias=None, overriding_path=None):
+    def declare(self, signer, contract, alias=None, max_fee=None, overriding_path=None):
         """Declare a smart contract class."""
-        return declare(contract, self.network, alias)
+        account = Account(signer, self.network)
+        return account.declare(
+            contract,
+            alias=alias,
+            max_fee=max_fee,
```

Comment:
Should we remove this from the NRE? We don't include `send` since it's already a method of `Account`. I'm thinking the same approach with `declare` should be fine i.e. `account.declare(...)`.